board/app/helpers/html_helper.php

```diff
@@ -1,7 +1,19 @@
 <?php
 
-function eh($string)
-{
+function eh($string){
     if (!isset($string)) return;
     echo htmlspecialchars($string, ENT_QUOTES);
+    }
+
+//View Modification (Line Break)
+function readable_text($s){
+    $s = htmlspecialchars($s, ENT_QUOTES);
+    $s = nl2br($s);
+    return $s;                    
+    }	
+
+function redirect($url){
```

Comment:
You can use the router from the app/config/router.php, as it is already declared in your bootstrap :bow: 
So you can remove this helper function already po :dart: